.github/workflows/pr-lint.yml

```diff
@@ -27,14 +27,10 @@ jobs:
       - name: Install
         run: yarn install --frozen-lockfile
 
-      - name: eslint
+      - name: eslint + stylelint
         continue-on-error: true
         id: eslint
-        run: yarn lint:es
-
-      - name: stylelint
-        continue-on-error: true
```

Comment:
if rm this, does it mean job will stop after detecting the first error? better continue to find all errors IMO :)

Reply:
not exactly, the command stylelint is remove because eslint and stylelint are grouped in one command who have also the continue-on-error.
So no change for this specific option

Reply:
Oh I missed that 😅 . Understood.